app/src/main/java/comcmput301f16t01/github/carrier/Requests/ElasticRequestController.java

```diff
@@ -129,8 +129,83 @@ protected RequestList doInBackground(String... search_parameters) {
+         */
+        private static final int DISTANCE = 50;
+
+        @Override
+        protected RequestList doInBackground(Location... search_parameters) {
+            verifySettings();
+
+            String query =
+                "{ \"from\" : 0, \"size\" : 500,\n" +
+                "  \"query\" : {\n" +
+                "    \"filtered\" : {\n" +
+                "      \"query\" : {\n" +
+                "        \"bool\": { " +
+                "          \"should\": [\n" +
+                "            { \"match\": { \"status\": 1 }},\n" +
+                "            { \"match\": { \"status\": 2 }}\n" +
+                "          ],\n" +
+                "          \"minimum_should_match\": \"1\"\n" +
+                "        }\n" +
+                "      },\n" +
+                "      \"filter\" : {\n" +
+                "        \"geo_distance\" : {\n" +
+                "          \"distance\" : \"" + DISTANCE + "km\",\n" +
+                "          \"location\" : [" + search_parameters[0].getLongitude() + ", "
+                                             + search_parameters[0].getLatitude() + "]\n" +
+                "        }\n" +
+                "      }\n" +
+                "    }\n" +
+                "  },\n" +
+                "  \"sort\": [ {\n" +
+                "    \"_geo_distance\": {\n" +
+                "      \"location\" : [" + search_parameters[0].getLongitude() + ", "
+                                         + search_parameters[0].getLatitude() + "],\n" +
+                "      \"order\": \"asc\",\n" +
+                "      \"unit\": \"km\",\n" +
+                "      \"distance_type\": \"plane\"\n" +
+                "    }\n" +
+                "  } ]\n" +
+                "}";
+
+            Search search = new Search.Builder(query)
+                    .addIndex("cmput301f16t01")
+                    .addType("request")
+                    .build();
+
+            RequestList foundRequests = new RequestList();
+
+            try {
+                SearchResult result = client.execute(search);
+                Log.i("Result", result.toString());
+                if (result.isSucceeded()) {
+                    List<Request> notificationList = result.getSourceAsObjectList(Request.class);
+                    foundRequests.addAll( notificationList );
+                } else {
+                    throw new IllegalArgumentException();
+                }
+            } catch (IOException e) {
+                e.printStackTrace();
+                Log.i("Error", "Something went wrong when we tried to talk to elastic search");
+            }
+            return foundRequests;
+        }
+    }
+
+    /**
+     * Get all of a rider's requests including a filter by status
+=======
      * Get all of a rider's requests filtered by status
      * @see RequestController#fetchRequestsWhereRider(User, Integer...)
+>>>>>>> 92fda80c294f949b664f49d206e3dedeea1f17a7
```

Comment:
You don't have to fix this, it's in #167, I think. 
I'll approve this branch when you merge that with it's changes?